src/pages/site/[uuid]/tabs/Overview.tsx

```diff
@@ -411,6 +412,21 @@ const SiteOverviewTab = ({ site, refetch: refetchEntity }: SiteOverviewTabProps)
                   value={site.combined_workday_count}
                 />
                 <GoalProgressCard label={t("Hectares Restored Goal")} value={site.hectares_to_restore_goal} />
+                <If condition={site.framework === Framework.PPC}>
+                  <Then>
+                    <GoalProgressCard
+                      label={t("Hectares Under Restoration")}
+                      value={site.total_hectares_restored_sum}
+                    />
+                  </Then>
+                  <Else>
+                    <GoalProgressCard
+                      label={t("Hectares Under Restoration")}
+                      value={site.total_hectares_restored_sum}
+                      limit={site.hectares_to_restore_goal}
+                    />
```

Comment:
on a second view of this, it is exactly the same component with just one extra parameter, please just add that param conditionally or make sure it is null when it is PPC